### PR TITLE
Mainline-poc update readme to V2

### PR DESCRIPTION
**Pull request by ommirzaei**
Description:

This Pull Request updates the examples in the README file to align with Marqo V2.

Note:

- This is a follow-up PR for https://github.com/marqo-ai/marqo/pull/672
- Marqo V2 supports both structured and unstructured indexes. By default all the indexes that we create in README are unstructured indexes to keep `Quick start` simple. 
- Currently creating index using `-` in its name e.g. `my-first-index` is failing. However, this will be fixed in V2. So, no need to update the README file. 
- Once this PR is reviewed/merged, will apply the same changes to the `py-marqo` project's README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Vector databases are specialized components for vector similarity and only servi
 
 Here is a code snippet for a minimal example of vector search with Marqo (see [Getting Started](#getting-started)):
 
-1. Use docker to run Marqo (Mac users with M-series chips will need to [go here](#m-series-mac-users)):
```
Comment (ommirzaei) [inline README.md:32]: With V2 Mac users will be able to use the same image that includes Vespa

File: README.md [hunk 2]
```diff
@@ -276,16 +273,6 @@ result = mq.index("my-first-index").search('marco polo', search_method=marqo.Sea
 
 ```
 
-### Search specific fields
```
Comment (ommirzaei) [inline README.md:279]: Since `searchable_attributes` hasn't been supported yet with V2, I removed the whole section of `Search specific fields`

File: README.md [hunk 3]
```diff
@@ -67,12 +67,11 @@ mq.index("my-first-index").add_documents([
                        "mobility, life support, and communications for astronauts",
         "_id": "article_591"
     }],
-    tensor_fields=["Title", "Description"],
-    auto_refresh=True
+    tensor_fields=["Title", "Description"]
```
Comment (pandu-k) [inline README.md]: we should only have a single tensor field - because this results in the best user expereience for unstructured indexes

File: README.md [hunk 4]
```diff
@@ -187,20 +186,18 @@ mq.index("my-first-index").add_documents([
...
```
Comment (pandu-k) [inline README.md]: ditto RE 1 tensor fields

File: README.md [hunk 5]
```diff
@@ -308,7 +295,7 @@ response = mq.index("my-multimodal-index").add_documents([{
     "My Image": "https://raw.githubusercontent.com/marqo-ai/marqo-api-tests/mainline/assets/ai_hippo_realistic.png",
     "Description": "The hippopotamus, also called the common hippopotamus or river hippopotamus, is a large semiaquatic mammal native to sub-Saharan Africa",
     "_id": "hippo-facts"
-}], tensor_fields=["My Image", "Description"], auto_refresh=True)
+}], tensor_fields=["My Image", "Description"])
```
Comment (pandu-k) [inline README.md]: ditto RE 1 tensor fields

File: README.md [hunk 6]
```diff
@@ -318,13 +305,6 @@ You can then search using text as usual. Both text and image fields will be sear
```
Comment (pandu-k) [inline README.md:305]: For the lines above - because we are only dealing with a single tensor field - the above text won't make as much sense

File: README.md [hunk 7]
```diff
@@ -369,8 +349,7 @@ mq.index("my-weighted-query-index").add_documents(
             "The last known of its species died in 1936.",
         },
     ],
-    tensor_fields=["Title", "Description"],
-    auto_refresh=True
+    tensor_fields=["Title", "Description"]
```
Comment (pandu-k) [inline README.md]: ditto RE 1 tensor fields

File: README.md [hunk 8]
```diff
@@ -425,24 +400,18 @@ mq.index("my-first-multimodal-index").add_documents(
...
             "Title": "Horse Jumping",
-            "captioned_image": {
-                "caption": "A person riding a horse over a jump in a competition.",
-                "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg",
-            },
+            "caption": "A person riding a horse over a jump in a competition.",
+            "image": "https://raw.githubusercontent.com/marqo-ai/marqo/mainline/examples/ImageSearchGuide/data/image1.jpg",
         },
     ],
     # Create the mappings, here we define our captioned_image mapping
```
Comment (pandu-k) [inline README.md:417]: have we tested this example?

File: README.md [hunk 9]
```diff
@@ -459,14 +428,12 @@ mq.index("my-first-multimodal-index").add_documents(
     },
     # We specify which fields to create vectors for. 
     # Note that captioned_image is treated as a single field.
-    tensor_fields=["Title", "captioned_image"],
-    auto_refresh=True
+    tensor_fields=["Title", "captioned_image"]
```
Comment (pandu-k) [inline README.md]: ditto RE 1 tensor fields

File: README.md (hunk 6 above)
Comment (ommirzaei) [reply README.md:305]: Are you suggesting to remove the sentence `Both text and image fields will be searched:`?

File: README.md (hunk 8 above)
Comment (ommirzaei) [reply README.md:417]: The query is working and returning response. However, since we have only one tensor filed, I think the tensor field needs to be `image`  field to make more sense, right?

File: README.md (hunk 6 above)
Comment (pandu-k) [reply README.md:305]: yep - because it is no longer true

File: README.md (hunk 8 above)
Comment (ommirzaei) [reply README.md:417]: Have checked this with Farshid, and he confirmed using `tensor_fields=["captioned_image"]` as tensor field.

File: README.md [hunk 10]
```diff
@@ -308,23 +295,16 @@ response = mq.index("my-multimodal-index").add_documents([{
     "My Image": "https://raw.githubusercontent.com/marqo-ai/marqo-api-tests/mainline/assets/ai_hippo_realistic.png",
     "Description": "The hippopotamus, also called the common hippopotamus or river hippopotamus, is a large semiaquatic mammal native to sub-Saharan Africa",
     "_id": "hippo-facts"
-}], tensor_fields=["My Image", "Description"], auto_refresh=True)
+}], tensor_fields=["My Image"])
 
 ```
 
-You can then search using text as usual. Both text and image fields will be searched:
+You can then search using text as usual.
```
Comment (pandu-k) [inline README.md]: Let's reword this to `You can then search the image field using text.`

Comment (pandu-k) [reply README.md]: As it makes more sense in this new context

File: README.md [hunk 11]
```diff
@@ -459,14 +428,12 @@ mq.index("my-first-multimodal-index").add_documents(
...
     # We specify which fields to create vectors for. 
     # Note that captioned_image is treated as a single field.
-    tensor_fields=["Title", "captioned_image"],
-    auto_refresh=True
+    tensor_fields=["captioned_image"]
 )
 
 # Search this index with a simple text query
 results = mq.index("my-first-multimodal-index").search(
     q="Give me some images of vehicles and modes of transport. I am especially interested in air travel and commercial aeroplanes.",
```
Comment (pandu-k) [inline README.md]: delete the comma at the end of this line

File: README.md [hunk 12]
```diff
@@ -478,14 +445,12 @@ results = mq.index("my-first-multimodal-index").search(
         "What are some vehicles and modes of transport?": 1.0,
         "Aeroplanes and other things that fly": -1.0,
     },
```
Comment (pandu-k) [inline README.md:447]: delete the comma at the end of the line

File: README.md [hunk 13]
```diff
@@ -478,14 +445,12 @@ results = mq.index("my-first-multimodal-index").search(
...
-    searchable_attributes=["captioned_image"],
 )
 print("\nQuery 2:")
 pprint.pprint(results)
 
 results = mq.index("my-first-multimodal-index").search(
     q={"Animals of the Perissodactyla order": -1.0},
```
Comment (pandu-k) [inline README.md]: delete the comma at the end of the line

Comment (ommirzaei) [reply README.md]: Thanks Pandu for the comments. Also, deleted the unnecessary commas in other places

**Review by pandu-k — APPROVED** [review]
